src/pages/start-page/components/MeetingLinkItem.tsx

```diff
@@ -19,7 +19,7 @@ const MeetingLinkItem = ({ meeting, removeMeeting }: IProps) => {
 
   const copyCodeToClipboard = () => {
     if (!isCopied) {
-      navigator.clipboard.writeText(`${window.location.origin}/join/${code}`);
+      navigator.clipboard.writeText(`${code}`);
```

Comment:
There is not need to use template string

use `navigator.clipboard.writeText(code);`  instead